README.md

```diff
@@ -164,3 +184,42 @@ Dieser Leitfaden ist in verschiedene Module aufgeteilt, die durch die 3 Jahre ei
 ## Extreme Programming
 ## Behaviour Driven Development
 ## Feature Driven Development
+
+# Konzepte
+## RAD - Rapid Application Development
+## MVC - Model View Controler
+
+
+# iOS - Lernreihenfolge
```

Comment:
sehr viele interessante dinge dabei, generell kann man da einiges in den Bereich UIKit rein schieben. Dies soll aber bewusst kein Leitfaden werden, der sich zu stark mit dem runter programmieren von einem "Tutorial" ist.